test/feature-benchmark/scenarios.py

```diff
@@ -803,3 +803,63 @@ class KafkaRecovery(KafkaScenario):
 true
 """
     )
+
+
+class KafkaRecoveryBig(Kafka):
+    SHARED = [
+        Td("$ kafka-create-topic topic=kafka-recovery-big partitions=8"),
+        # Ingest 10M records
+        Kgen(
+            topic="kafka-recovery-big",
+            args=[
+                "--keys=random",
+                "--num-records=10000000",
+                "--values=bytes",
+                "--max-message-size=32",
+                "--min-message-size=32",
+                "--key-min=256",
+                "--key-max=10000000",
```

Comment:
this will essentially guarantee key duplicates and make it so the actual number of records persisted is < 10 million because some duplicates will get cancelled out in the upsert operator itself. I would recommend using a key range > 100 trillion (10 million squared) or using sequential and filtering on the values instead?

Reply:
woke up with the realization that you want duplicates in some cases to test the performance with respect to compaction of the underlying data, so perhaps the best thing would be if this could somehow be a command line argument? no worries if thats not easy atm though

Reply:
Unfortunately I am missing some pieces that are needed to make this a command-line argument, however if you can suggest reasonable defaults in the meantime that would be much appreciated!

Reply:
1 trillion seems like a nice, reasonable default (I pulled this number out of a hat because it intuitively seems like it would have a very low, but nonzero percentage of duplicates)